src/helper.js

```diff
@@ -1,10 +1,29 @@
 var path = require("path");
+var os = require("os");
 
 module.exports = {
 	HOME: (process.env.HOME || process.env.USERPROFILE) + "/.lounge",
-	getConfig: getConfig
+	getConfig: getConfig,
+	expandHome: expandHome,
 };
 
 function getConfig() {
 	return require(path.resolve(this.HOME) + "/config");
 }
+
+function expandHome(path) {
+	var home;
+
+	if (os.homedir) {
+		home = os.homedir();
+	}
+
+	if (!home) {
+		home = process.env.HOME || "";
+	}
+
+	home = home.replace("$", "$$$$");
```

Comment:
Why is this necessary? Escaping? Why 4 of them lol? Would have `\$` work too (fairly common when escaping stuff)?